docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -536,6 +537,22 @@ Specifies your computer's hostname. It is used when running tests in [remote bro
 
 If the hostname is not specified, TestCafe uses the operating system hostname or network IP address of the current machine.
 
+### --ssl \<options\>
+
+Provides options that allow you to establish HTTPS connection between the client browser and the TestCafe server.
+
+The `options` parameter contains options required to initialize
+[a Node.js HTTPS server](https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener).
```

Comment:
Should we use [this](https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options) link instead?

Reply:
No. This link is right.
You can specify all options that the build-in `https.createServer` method can accept.

Reply:
After discussing in private we decide to add he following phrase:
```js
See description of the mostly used ssl options in https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options
```